src/counts.jl

```diff
@@ -283,6 +283,40 @@ function addcounts_dict!(cm::Dict{T}, x::AbstractArray{T}) where T
     return cm
 end
 
+"""Specialist addcounts methods for small bits types"""
```

Comment:
This docstring doesn't really add anything given that the signature already gives this information. Also, docstrings are attached to a given method, so you cannot use the plural here. Better make this a standard comment, with a line break after it so that it's clear it describes several functions and not just this one.